code/modules/virus2/dishincubator.dm

```diff
@@ -48,7 +48,7 @@
 			scancount += SP.rating-1
```

Comment:
Assuming tiers are 1 2 3 rating, you'd get 1x 2x 3x mutatechance instead of currently with 0x 1x 2x